v2/internal/frontend/desktop/darwin/inspector_dev.go

```diff
@@ -1,8 +1,8 @@
-//go:build darwin && dev
+//go:build darwin && (dev || debug)
 
 package darwin
 
-// We are using private APIs here, make sure this is only included in a dev build and not in a production build.
+// We are using private APIs here, make sure this is only included in a dev/debug build and not in a production build.
```

Comment:
Should we output this as a warning when compiling with debug?

Reply:
Yeah we can do that. I'll wait until the cli refactor has been merged so we don't get conflicts.

Reply:
Merged

Reply:
Added the warning, what do you think?